personalized_active_learning/datasets/types.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+import numpy as np
+import torch
+
+
+@dataclass()
+class TextFeaturesBatch:
```

Comment:
[**Question**] Could you add more specific docstring with information: How to add new definitions and what are the use cases? Honestly, I am little confused here (and I don't remember how to use it :) )

Reply:
Can you elaborate? I am not sure about which `definitions` and `use cases` you have in mind. :stuck_out_tongue:

Reply:
Just add comment what is it for :laughing:

Reply:
Oh, sure xd. Added:p